index.html

```diff
@@ -35,6 +35,11 @@
     <div class="home-map" data-hash-association="#!/home" class="active"></div>
 
     <div class="bat-detective app">
+      <div class="world-tour-banner">
+        <h2>Bat Detective World Tour starts November 2 with the United Kingdom!</h2>
+        <p>Learn more at the <a href="http://blog.batdetective.org/" target="_blank">blog</a></p>
```

Comment:
Presumably the link should now be http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/. Still waiting on final wording for the text.